src/parseTorrent.ts

```diff
@@ -23,9 +25,78 @@ interface Torrent {
 
 		private: number;
 	};
-	comment: Buffer | string;
-	announce: Buffer;
-	"announce-list": Buffer[][];
+	// qBittorrent only keeps info dict, everything else in fastresume
+	comment?: Buffer | string;
+	announce?: Buffer;
+	"announce-list"?: Buffer[][];
+}
+
+/**
+ * "Fastresume" for all clients. Used to get torrent info.
+ * qBittorrent: .fastresume, Transmission: .resume, rTorrent: .rtorrent
+ * Deluge doesn't store labels in fastresume, but in label.conf
+ */
+interface TorrentMetadata {
+	trackers?: Buffer[][];
+	"qBt-category"?: Buffer;
+	"qBt-tags"?: Buffer;
+	custom1?: Buffer;
+	label?: Buffer;
+	labels?: Buffer;
```

Comment:
is Buffer the right type for these properties?

Reply:
Yes, these should be all encoded similar to .torrent files. Though only qbit is using it currently.

Reply:
if only qBit is using it, the `custom1`/`label`/`labels` properties don't need to be there.